site/source/tutorials/001_static_content/demo/finished/biz-e-corp/tests/unit/widgets/HelloWorld.ts

```diff
@@ -9,7 +9,8 @@ registerSuite({
 		const helloWorld = new HelloWorld();
 
 		const vnode = <VNode> helloWorld.__render__();
-		assert.strictEqual(vnode.vnodeSelector, 'div');
-		assert.equal(vnode.text, 'Hello, Dojo World!');
+		assert.strictEqual(vnode.vnodeSelector, 'h1');
+		assert.equal(vnode.text, 'Biz-E Bodies');
+		assert.equal(vnode.properties.title, 'I am a title!');
```

Comment:
For tests you can do, `vnode.properties!.title` (the `!` asks TS to ignore the strict null check)